packages/effect/src/RateLimiter.ts

```diff
@@ -0,0 +1,11 @@
+import type { Effect } from "./Effect.js"
```

Comment:
I wasn't quite sure which package to put this under, so its in the `effect` package namespace for now.

Reply:
Yeah that's correct